rootfs/etc/nginx/template/nginx.tmpl

```diff
@@ -841,6 +841,7 @@ stream {
 
             proxy_pass_request_body     off;
             proxy_set_header            Content-Length "";
+            proxy_set_header            X-Fowarded-Proto "";
```

Comment:
@wayt I think you have a typo here, it should be `X-Forwarded-Proto`

Reply:
.... yup